config/routes.rb

```diff
@@ -2,6 +2,12 @@
   devise_for :users
 
   namespace :editorial do
+    get '/news' => 'news#index', as: :news_articles
```

Comment:
So the issue is that resources creates `editorial_news_path` helpers, and the simple_form_for method uses the object type (news_article) to try to map paths to `editorial_news_articles_path`. Perhaps there's a way to specify the URL better in the form.